addons/website_forum/views/forum_post_views.xml

```diff
@@ -24,14 +24,14 @@
                         <field name="active" invisible="1"/>
                         <field name="forum_id"/>
                         <field name="website_id" groups="website.group_multi_website"/>
-                        <field name="parent_id"/>
+                        <field name="parent_id" attrs="{'invisible':[('parent_id','=',False)]}"/>
```

Comment:
thanks @flch-odoo , updated the pr .
in case if you have time, can you review this PR's related to forum: 

* https://github.com/odoo/odoo/pull/127023
* https://github.com/odoo/odoo/pull/123636
* https://github.com/odoo/odoo/pull/129077
* https://github.com/odoo/odoo/pull/118686